html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Form</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <div>
+      <label for="email">Email:<br /></label>
+      <input
+        type="email"
+        id="email"
+        pattern=".+@globex\.com"
+        size="30"
+        required />
+    </div>
+    <div>
+      <label for="responses">Why is coding cool?<br /></label>
+      <select name="responses">
+        <option value="default">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <div>
+      <label for="languages">Favorite language? (choose one)<br /></label>
```

Comment:
for the radio buttons, you will either need a label for each one with matching for and id attributes, or you can wrap each input with a label. Functionality is the same in which you can click on the label and activate the associated input.